go.mod

```diff
@@ -36,7 +36,7 @@ require (
 	github.com/hashicorp/go-msgpack v0.0.0-20150518234257-fa3f63826f7c // indirect
 	github.com/hashicorp/raft v1.0.0 // indirect
 	github.com/hashicorp/vault/api v1.0.2
-	github.com/influxdata/flux v0.47.0
+	github.com/influxdata/flux v0.47.1-0.20190917165150-71861fd0953f
```

Comment:
This currently points to flux@feat/inject.

Once we will release v0.48 this will change